src/findings_data_import.py

```diff
@@ -20,6 +21,9 @@
 
 SUCCEEDED_FOLDER = "success"
 FAILED_FOLDER = "failed"
+# Identifier for V1 vs V2 schema
+V1_SCHEMA = "v1"
+V2_SCHEMA = "v2"
```

Comment:
This is due to the other suggestions regarding schema fields.

```suggestion
```

Reply:
We're not gonna change this at this time. If it is an issue I suggest we do it as part of a larger overhaul that covers how V1 and V2 *should* work, or after usage for a while we just rip out any "V1" style operations at all since this schema should be phasing out.

Reply:
Only the person who originally created the conversation (or someone that person directed) should be resolving conversations.